inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/beans/ClusterBean.java

```diff
@@ -43,4 +43,10 @@
     @Value("${sort.appName}")
     private String appName;
 
+    @Value("${pulsar.admin}")
+    private String pulsarAdmin;
+
+    @Value("${pulsar.service}")
+    private String pulsarService;
```

Comment:
It's better to rename these two parameters with `url` suffix like `pulsarAdminUrl` and `pulsarServiceUrl`